<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Segment</class>
 <widget class="qSlicerWidget" name="Segment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="SegmentCollapsibleGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>A) Segment liver</string>
     </property>
     <property name="collapsedHeight">
      <number>5</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="SegmentInputNodeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Input image volume</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="SegmentInputNodeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the fixed image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Fixed</string>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Segmented liver volume</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>0</string>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="SegmentOutputNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>0</string>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="SegmentGoToModulePushButton">
          <property name="toolTip">
           <string>Open the &quot;BrainsFit&quot; module</string>
          </property>
          <property name="text">
           <string>Go to Editor module</string>
          </property>
          <property name="workflow" stdset="0">
           <stringlist>
            <string>2</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="SegmentSaveToolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>0</string>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Segment</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SegmentInputNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Segment</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SegmentOutputNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
